content/blog/2022/03/admission-policy.md

```diff
@@ -0,0 +1,89 @@
+---
+title: Introducing the AdmissionPolicy
+authors:
+- Raul Cabello Martin
+date: 2022-03-09
+---
+
+Up until now, the only way to define a policy in Kubewarden was to use a `ClusterAdmissionPolicy`. `ClusterAdmissionPolicies` are applied to matching cluster-wide resources and resources across all namespaces.
+
+We are thrilled to announce `AdmissionPolicy`, a namespaced resource. The policy will process only the requests that are targeting the namespace where the `AdmissionPolicy` is defined. Other than that, there are no functional differences between `AdmissionPolicy` and `ClusterAdmissionPolicy`.
+
+## Understanding the motivation for using AdmissionPolicies
+
+It is possible to restrict the namespaces where a `ClusterAdmissionPolicy` evaluates resources using a `namespaceSelector`, but there is no way for kubernetes administrators to restrict users from creating `ClusterAdmissionPolicies` that evaluate resources just in a particular namespace. 
+
+Allowing all tenants to deploy `ClusterAdmissionPolicies` is risky. A tenant could apply policies that affect resources in all namespaces, even if he doesn't have access to all of them.
+You probably want to allow tenants to deploy policies just in the namespaces they have access to. That's where `AdmissionPolicies` are useful! We can achieve this restriction with `AdmissionPolicies` and Kubernetes’s Role-Based Access Control (RBAC).
+
+Imagine we have a tenant who can deploy resources just to the development namespace. We can allow this tenant to deploy `AdmissionPolicies` just in the development namespace using RBAC. If we allowed him to create `ClusterAdmissionPolicies`, he could block other resources from being created in other namespaces.
+
+## Let's see an AdmissionPolicy in Action!
+
+We are going to create an AdmissionPolicy that rejects privileged pods from being created in the production namespace.
+Assume we have a kubernetes cluster with Kubewarden installed. Follow the [quick start guide](https://docs.kubewarden.io/quick-start.html) if you don't have Kubewarden installed.
```

Comment:
```suggestion
Assume we have a Kubernetes cluster with Kubewarden installed. Follow the [quick start guide](https://docs.kubewarden.io/quick-start.html) if you don't have Kubewarden installed.
```

Reply:
done in https://github.com/kubewarden/kubewarden.io/pull/64/commits/8398829928bc5cc1b36718060fb70a2f84b024cd